tests/src/services/payment.service.test.ts

```diff
@@ -250,36 +309,36 @@ describe('Payments Service tests', () => {
         );
 
         jest
-          .spyOn(stripe.invoices, 'create')
+          .spyOn(stripeNewVersion.invoices, 'create')
           .mockResolvedValue(mockedInvoice as unknown as Stripe.Response<Stripe.Invoice>);
         jest
-          .spyOn(stripe.invoiceItems, 'create')
+          .spyOn(stripeNewVersion.invoiceItems, 'create')
           .mockResolvedValue(mockedInvoice.lines.data[0] as unknown as Stripe.Response<Stripe.InvoiceItem>);
 
-        jest.spyOn(stripe.invoices, 'update').mockImplementation();
+        jest.spyOn(stripeNewVersion.invoices, 'update').mockImplementation();
         jest
-          .spyOn(stripe.invoices, 'retrieve')
+          .spyOn(stripeNewVersion.invoices, 'retrieve')
```

Comment:
This spyOn does not work correctly; it throws an error indicating that the fake invoice ID does not exist. It is as if it were searching Stripe as such, instead of using the simulated value in spyOn.